Gemfile

```diff
@@ -3,29 +3,30 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { github: "CodiTramuntana/decidim", branch: "release/0.27-stable" }.freeze
+DECIDIM_VERSION = { github: "CodiTramuntana/decidim", branch: "release/0.28-stable" }.freeze
 
 gem "decidim", DECIDIM_VERSION
 gem "decidim-conferences", DECIDIM_VERSION
-gem "decidim-file_authorization_handler", github: "CodiTramuntana/decidim-file_authorization_handler", tag: "v0.27.1.7"
 gem "decidim-initiatives", DECIDIM_VERSION
 gem "decidim-templates", DECIDIM_VERSION
 
-gem "decidim-decidim_awesome", github: "decidim-ice/decidim-module-decidim_awesome", branch: "main"
+gem "decidim-decidim_awesome", "~> 0.11.4"
+gem "decidim-file_authorization_handler", github: "CodiTramuntana/decidim-file_authorization_handler", branch: "master"
 # Simplified & mobile-first proposals creation (ie: fixmystreets behavior)
-# gem "decidim-reporting_proposals", "~> 0.5.0"
-gem "decidim-reporting_proposals", github: "openpoke/decidim-module-reporting-proposals", branch: "release/0.27-stable"
+gem "decidim-reporting_proposals", github: "CodiTramuntana/decidim-module-reporting-proposals", branch: "fix/error_with_form_builder_override"
 # VALiD & ViaOberta integration
 gem "decidim-trusted_ids", github: "ConsorciAOC-PRJ/decidim-module-trusted-ids", branch: "main"
 
 gem "decidim-cdtb"
 
-gem "base64", "0.1.0"
-gem "net-smtp", "~> 0.4.0"
-gem "strscan", "3.0.0"
+gem "base64", "0.1.1"
+gem "net-smtp"
+gem "stringio", "3.0.1"
+gem "strscan", "3.0.1"
 
 gem "bootsnap", "~> 1.3"
 gem "wicked_pdf", "~> 2.7.0"
+gem "wkhtmltopdf-binary"
```

Comment:
why? isn't wkhtmltopdf already installed in the servers?

Reply:
I think this is a problem in the servers but I dont remember..

Reply:
Remove please. If its not in `main` it is not being needed now. And this library is required since the beginning of Decidim.